package.xml

```diff
@@ -30,14 +30,12 @@
 
   <url>https://github.com/googlecartographer/cartographer</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
-
   <build_depend>g++-static</build_depend>
   <build_depend>google-mock</build_depend>
   <build_depend>python-sphinx</build_depend>
 
   <depend>boost</depend>
-  <depend>ceres-solver</depend>
+  <depend>ceres_solver</depend>
```

Comment:
this change was actually not necessary we just needed to install the ceres-solver dependency from apt